lessons/Projects/project-beautiful-data/langs/en-us/index.adoc

```diff
@@ -0,0 +1,99 @@
+= Project: Beautiful Data
+
+@description{Students define row-consuming functions and combine them with advanced display contracts to create compelling data displays.}
+
+@lesson-prereqs{functions-examples-definitions}
```

Comment:
This can't be the right prereq. Kids don't even know how to do lookups! Shouldn't this be `advanced-displays`?